packages/ckeditor5-editor-multi-root/src/multirooteditor.ts

```diff
@@ -602,6 +708,12 @@ export type DetachRootEvent = {
  */
 export type AddRootOptions = {
 	data?: string;
+	attributes?: RootAttributes;
```

Comment:
In the doclet for `AddRootOptions` type, `@param` is used incorrectly. Object properties should be documented for each property in a separate doclet for each field.